docs/topics/wasm/wasm-debugging.md

```diff
@@ -0,0 +1,146 @@
+[//]: # (title: Debug Kotlin/Wasm code)
+
+> Kotlin/Wasm is in [Alpha](components-stability.md). It may be changed at any time.
+>
+{type="note"}
+
+This tutorial demonstrates how to debug a web application built with Kotlin/Wasm, based on a 
+[Compose Multiplatform](https://www.jetbrains.com/lp/compose-multiplatform/) example from our 
+`kotlin-wasm-examples` [repository](https://github.com/Kotlin/kotlin-wasm-examples/tree/main).
+
+## Before you start
+
+1. Download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/).
+2. Clone the [Kotlin/Wasm examples](https://github.com/Kotlin/kotlin-wasm-examples/) repository
+   by selecting **File** | **New** | **Project from Version Control** in IntelliJ IDEA.
+
+   You can also clone it from the command line:
+
+   ```bash
+   git clone git@github.com:Kotlin/kotlin-wasm-examples.git
+   ```
+
+> Alternatively, you can use our experimental [web wizard](https://kmp.jetbrains.com/) to download a sample project.
+>
+{type="note"}
+
+## Run the application
+
+1. In IntelliJ IDEA, a **Gradle build scripts found** notification appears. Click **Load**. 
+
+2. Open the **Gradle** tool window by selecting **View** | **Tool Windows** | **Gradle**.
+
+   > You need at least Java 11 as your Gradle JVM for the examples to load successfully.
+   >
+   {type="note"}
+
+3. In **compose-example** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserRun** task.
+
+   ![Run the Gradle task](wasm-gradle-task-window.png){width=650}
+
+    You can also run the following command in the terminal from the `compose-example` directory:
+
+   ```bash
+   ./gradlew wasmJsBrowserRun
+   ```
+   >Alternatively, you can load **compose-example** by opening it directly from the file explorer in **File | Open**.
+   >
+   {type="note"}
+
+4. Once the application starts, open the following URL in your browser:
+
+   ```bash
+   http://localhost:8080/
+   ```
+   
+   > The port number can vary. If the 8080 port is unavailable, you can find the corresponding port number printed in the console
```

Comment:
The actual port number could be different because 8080 is unavailable. You can find the actual port number printed in a Gradle build console.